uploading_packages/artifactory_ce.rst

```diff
@@ -72,7 +72,7 @@ This Python script might be helpful, given that it already defines the respectiv
     packages = output.splitlines()
 
     for package in packages:
-    print("Downloading %s" % package)
-    run("conan download %s -r=local" % package)
+        print("Downloading %s" % package)
+        run("conan download %s -r=local" % package)
 
-    run("conan upload * --all --confirm -r=artifactory")
\ No newline at end of file
+        run("conan upload * --all --confirm -r=artifactory")
```

Comment:
I think this for-loop was intended for the first 2 lines, bu the upload applies to *=all packages, so you don't need to iterate it.